### DISCUSSION TITLE
Welcome to Darkorbit-client Discussions!

### DISCUSSION THREAD

**Discussion by kaiserdj** [kaiserdj/Darkorbit-client#1] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Astrectt** [comment]
What is the purpose of auto login if you need to put master password and do a lot of steps? xd

**Reply by kaiserdj** [reply]
The purpose of the autologin, is to be able to access an unlimited number of accounts with only 1 password and that if you do not like the master password, you can use the automatic login from the launch parameter or by setting the master password to be empty, so you will not have to write anything to log in with your accounts. 

Likewise, it is a utility, you don't have to use it if you don't like it.

**Reply by mikefi2** [reply]
Hi guys! New to github. Try to run the darkorbit-client. I have installed the latest java given. run the npm install from the command promt. App started ok. I've logged in to my account. Any ideas on how to install your flash player? Thanks in advance. Mike

**Comment by spookmonkey92** [comment]
I have a question, and I am naive. I am hoping this is for the game Dark Orbit and its new client. I am trying to play with my best friends we all started as teenagers with the game. One friend stuck with it for 10 years or so, and we are starting back. They have windows. I have mac. The client Dark Orbit provides will not work. If I am in the right spot please help. For the sake of my sanity. Lol be glad to drop a few bucks for your time. Thank you.

**Reply by spookmonkey92** [reply]
Ok so I see, but how do I download it correctly for mac. I tried and says to contact my computer administrator. I have 2015 Macbook Pro.

**Reply by kaiserdj** [reply]
I don't really understand what is happening to you, this client is what you need if you want to play darkorbit from a Mac. If you need help, you can create an issue or join our discord server: https://discord.gg/vHqAaxS7xF

**Reply by hellt4ker** [reply]
Have you tried doing it why administrative privileges?
If it says to contact the computer administrator, you're likely running as simple user while it expects you to run it as superuser.

**Comment by kubajak** [comment]
Can I find somewhere background maps?

**Reply by kaiserdj** [reply]
what?

**Comment by vitmitul** [comment]
hey, would be possible to show enemy hp?

**Reply by kaiserdj** [reply]
This program is only a client, it is not a hack or something similar for darkorbit. I guess if you want to do that you will have to have third party programs like Darkbot

**Reply by vitmitul** [reply]
ok thanks

**Comment by hellt4ker** [comment]
Hey there, first of all I want to thank you for the great work. I only came back playing because I was curious in making the client work on my Linux Mint. It's been maybe a week now, and it works great.

Unfortunately, it seems to bug a bit when clicking the items on the hangar as I try to manage the ones on the drones. That's point one.
Since yesterday, I think as I upgraded npm, it started having this bug where my keyboard is rendered useless to use the slots and I can't manually click the items on them either. I can however attack with double-click, but not change ammunition or deactivate CPUs.

In fact, I tried reinstalling everything today - git, npm and the client. I still haven't tested.

npm returned me this:
```
npm WARN old lockfile 
npm WARN old lockfile The package-lock.json file was created with an old version of npm,
npm WARN old lockfile so supplemental metadata must be fetched from the registry.
npm WARN old lockfile 
npm WARN old lockfile This is a one-time fix-up, please be patient...
npm WARN old lockfile 
npm WARN EBADENGINE Unsupported engine { package: 'builder-util-runtime@8.7.5',
npm WARN EBADENGINE   required: { node: '>=12.0.0' },
npm WARN EBADENGINE   current: { node: 'v10.19.0', npm: '7.21.1' } }
npm WARN EBADENGINE Unsupported engine { package: 'fs-extra@10.0.0',
npm WARN EBADENGINE   required: { node: '>=12' },
npm WARN EBADENGINE   current: { node: 'v10.19.0', npm: '7.21.1' } }
npm WARN EBADENGINE Unsupported engine { package: 'open@8.2.1',
npm WARN EBADENGINE   required: { node: '>=12' },
npm WARN EBADENGINE   current: { node: 'v10.19.0', npm: '7.21.1' } }
npm WARN EBADENGINE Unsupported engine { package: 'fs-extra@10.0.0',
npm WARN EBADENGINE   required: { node: '>=12' },
npm WARN EBADENGINE   current: { node: 'v10.19.0', npm: '7.21.1' } }
npm WARN EBADENGINE Unsupported engine { package: 'yargs@17.0.1',
npm WARN EBADENGINE   required: { node: '>=12' },
npm WARN EBADENGINE   current: { node: 'v10.19.0', npm: '7.21.1' } }
npm WARN deprecated uuid@3.4.0: Please upgrade  to version 7 or higher.  Older versions may use Math.random() in certain circumstances, which is known to be problematic.  See https://v8.dev/blog/math-random for details.

> darkorbit-client@2.5.1 postinstall
> electron-builder install-app-deps

  • electron-builder  version=22.10.5
  • loaded configuration  file=package.json ("build" field)

added 382 packages, and audited 383 packages in 3m

39 packages are looking for funding
  run `npm fund` for details

2 moderate severity vulnerabilities

To address all issues, run:
  npm audit fix

Run `npm audit` for details.
```

After running 'npm audit':
```
npm WARN EBADENGINE Unsupported engine { package: 'builder-util-runtime@8.7.5',
npm WARN EBADENGINE   required: { node: '>=12.0.0' },
npm WARN EBADENGINE   current: { node: 'v10.19.0', npm: '7.21.1' } }
npm WARN EBADENGINE Unsupported engine { package: 'fs-extra@10.0.0',
npm WARN EBADENGINE   required: { node: '>=12' },
npm WARN EBADENGINE   current: { node: 'v10.19.0', npm: '7.21.1' } }
npm WARN EBADENGINE Unsupported engine { package: 'open@8.2.1',
npm WARN EBADENGINE   required: { node: '>=12' },
npm WARN EBADENGINE   current: { node: 'v10.19.0', npm: '7.21.1' } }
npm WARN EBADENGINE Unsupported engine { package: 'fs-extra@10.0.0',
npm WARN EBADENGINE   required: { node: '>=12' },
npm WARN EBADENGINE   current: { node: 'v10.19.0', npm: '7.21.1' } }
npm WARN EBADENGINE Unsupported engine { package: 'yargs@17.0.1',
npm WARN EBADENGINE   required: { node: '>=12' },
npm WARN EBADENGINE   current: { node: 'v10.19.0', npm: '7.21.1' } }

up to date, audited 383 packages in 3s

39 packages are looking for funding
  run `npm fund` for details

found 0 vulnerabilities
```

I also tried 'npm audit' yesterday before the playing with similar results. It seems you have to recompile the whole of the client with the newer versions and update it here on github. I'll try it again today and if it doesn't work I'll try downgrading npm, test and have another feedback posted here. By the way, I didn't quite know where to post this. Any suggestions would be great too.
Cheers.

**Reply by kaiserdj** [reply]
Hi, thank you very much for your comment.
Do you still have the same problem?
If you still have it, please create an issue so that it can help you better
